cvat-core/src/session.ts

```diff
@@ -567,6 +571,22 @@ export class Job extends Session {
                 updatedDate: {
                     get: () => data.updated_date,
                 },
+                sourceStorage: {
+                    get: () => (
+                        new Storage({
+                            location: data.source_storage?.location || StorageLocation.LOCAL,
+                            cloudStorageId: data.source_storage?.cloud_storage_id,
+                        })
+                    ),
+                },
+                targetStorage: {
+                    get: () => (
+                        new Storage({
+                            location: data.target_storage?.location || StorageLocation.LOCAL,
+                            cloudStorageId: data.target_storage?.cloud_storage_id,
+                        })
+                    ),
+                },
```

Comment:
Do you think we need to create new instance each time on get? 
It leads to strange behavior, like ``Job.targetStorage !== Job.targetStorage``